packages/ui/src/composites/wui-tabs/index.ts

```diff
@@ -50,7 +50,7 @@ export class WuiTabs extends LitElement {
       this.buttons = [...this.shadowRoot.querySelectorAll('button')]
       setTimeout(() => {
         this.animateTabs(0, true)
-      }, 1)
+      }, 0.1)
```

Comment:
You can change this to 0, setTimeout triggers next cpu tick, so it should work fine for what you are trying to acheive